examples/layers/raw/hello_world.dart

```diff
@@ -12,26 +12,37 @@ void beginFrame(Duration timeStamp) {
   final ui.Size logicalSize = ui.window.physicalSize / devicePixelRatio;
 
   final ui.ParagraphBuilder paragraphBuilder = ui.ParagraphBuilder(
-    ui.ParagraphStyle(textDirection: ui.TextDirection.ltr),
+    ui.ParagraphStyle(
+      textDirection: ui.TextDirection.ltr,
+      fontSize: 20,
+      fontWeight: ui.FontWeight.bold,
+    ),
```

Comment:
ideally for this example we would keep it as simple as possible and not set the styles. In theory it should end up with the exact same rendering as the equivalent widget version.